internal/pkg/skuba/kubeadm/configmap.go

```diff
@@ -133,11 +134,29 @@ func setApiserverAdmissionPlugins(initConfiguration *kubeadmapi.InitConfiguratio
 	if initConfiguration.APIServer.ControlPlaneComponent.ExtraArgs == nil {
 		initConfiguration.APIServer.ControlPlaneComponent.ExtraArgs = map[string]string{}
 	}
+	admissionPlugins := []string{}
+	if len(initConfiguration.APIServer.ExtraArgs["enable-admission-plugins"]) > 0 {
+		admissionPlugins = strings.Split(initConfiguration.APIServer.ExtraArgs["enable-admission-plugins"], ",")
```

Comment:
this looks like a good place for simple *unit-test*.

Reply:
Adding that one, yup!

Reply:
Adding that one, yup!